lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -237,8 +237,8 @@ LogicalResult ClassTypeOp::verify() {
 // PrimLoopOp
 //===----------------------------------------------------------------------===//
 
-OperandRange PrimLoopOp::getSuccessorEntryOperands(unsigned index) {
-  assert(index == 0);
+OperandRange PrimLoopOp::getSuccessorEntryOperands(::llvm::Optional<unsigned int> index) {
+  assert(index.hasValue() && index.value() == 0);
```

Comment:
cc @silvasean this was the only substantive patch.

Reply:
I would omit the ::llvm but seems fine to me.